src/BuiltInTools/dotnet-watch/Filters/BrowserRefreshServer.cs

```diff
@@ -100,22 +100,25 @@ public ValueTask SendJsonSerlialized<TValue>(TValue value, CancellationToken can
 
         public async ValueTask SendMessage(ReadOnlyMemory<byte> messageBytes, CancellationToken cancellationToken = default)
         {
-            for (var i = 0; i < _clientSockets.Count; i++)
+            try
             {
-                var clientSocket = _clientSockets[i];
-                if (clientSocket.CloseStatus.HasValue)
-                {
-                    continue;
-                }
-
-                try
+                for (var i = 0; i < _clientSockets.Count; i++)
                 {
+                    var clientSocket = _clientSockets[i];
+                    if (clientSocket.CloseStatus.HasValue)
+                    {
+                        continue;
+                    }
                     await clientSocket.SendAsync(messageBytes, WebSocketMessageType.Text, endOfMessage: true, cancellationToken);
                 }
-                catch (Exception ex)
-                {
-                    _reporter.Verbose($"Refresh server error: {ex}");
-                }
+            }
+            catch (TaskCanceledException)
```

Comment:
Added this because I noticed we weren't handling the scenario where someone `Ctrl+C` the watch server.

I moved the try-catch outside the for-loop because I noticed it would spew N logs for every N clients connected to the browser server and that seemed excessive.

Can a